tests/suites/test_suite_psa_crypto.data

```diff
@@ -503,6 +503,170 @@ PSA import/export RSA keypair: import PEM
+
+PSA import/export FFDH RFC7919 2048 key pair: export not permiterd
+depends_on:PSA_WANT_ALG_FFDH:PSA_WANT_KEY_TYPE_FFDH_KEY_PAIR:MBEDTLS_PK_PARSE_C:MBEDTLS_PK_WRITE_C
+import_export:"2A45292441157B3C25572F76A5CDF960A7BDBF06731D783C5BF8920FB94CCC3D5DCCF86A3CB66B4E3AEDD23106222458ACF3F72C753CB67C2E19AD399566866FEBC16C3B4DC72773B4709047AE1AEC2D9107C2041B06B86A8F604465B26E0E753D6B10772798B3797232D950A36F2D4B33B04B36DE73AC6B8A7365015DF5745A1F892728B0CA947702C36E3BC646E72E23E80C345DBB014B7F93B36C80B4051F9A716D19B980861E86D62977466565462FBD3C1BB4EFD630DCCBEB351A7FA95602B7FE23903C7C7DC999950493BEC028AC42346858FAD969452DCF1DE9AD445F7F928D63B75FA86E8C1D722AB242D91995D3545A1791D72B0F384E74B45C7C01":PSA_KEY_TYPE_DH_KEY_PAIR(PSA_DH_FAMILY_RFC7919):0:PSA_ALG_FFDH:0:2048:0:PSA_ERROR_NOT_PERMITTED:1
+
+PSA import/export FFDH RFC7919 2040 key pair: import invalid kay length
```

Comment:
```suggestion
PSA import/export FFDH RFC7919 2040 key pair: import invalid key length
```
I think this should be fixed also for the tests below

Reply:
I will fix this when more review comments will be provided.